libarchive/archive_write_set_format_mtree.c

```diff
@@ -862,6 +863,40 @@ mtree_entry_free(struct mtree_entry *me)
 	free(me);
 }
 
+static void
+mtree_copy_ae_digests(struct reg_info *reg, struct archive_entry *entry, int compute_sum)
+{
+	reg->compute_sum = compute_sum;
+	if ((reg->compute_sum & F_MD5) && entry->mset_digest.md5) {
+		memcpy(&reg->digest.md5, entry->digest.md5, sizeof(reg->digest.md5));
+                reg->mset_digest.md5 = entry->mset_digest.md5;
+	}
+	if (reg->compute_sum & F_RMD160 && entry->mset_digest.rmd160) {
```

Comment:
For clarity, please parenthesize this (and the following) the same way you did the first MD5 test above.

Reply:
done.